Dockerfile

```diff
@@ -0,0 +1,60 @@
+# Use this with
+#
+#  docker build -t evm_rpc .
+#  or use ./scripts/docker-build
+#
+# The docker image. To update, run `docker pull ubuntu` locally, and update the
+# sha256:... accordingly.
+FROM ubuntu@sha256:626ffe58f6e7566e00254b638eb7e0f3b11d4da9675088f4781a50ae288f3322 as deps
+
+ENV TZ=UTC
+
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone && \
+    apt -yq update && \
+    apt -yqq install --no-install-recommends curl ca-certificates \
+        build-essential pkg-config libssl-dev llvm-dev liblmdb-dev clang cmake
```

Comment:
This comes from the [Internet Identity Dockerfile](https://github.com/dfinity/internet-identity/blob/83b5732bccee9f53dc81f4658749aac6a8361000/Dockerfile#L12-L15). I'm curious whether this is reproducible if one of these packages is updated on the `apt` repository. Does anyone have any insights about this?

Reply:
We never ran into issues so far, but good point. It seems broken?
@nmattia: did we just get lucky (or didn't notice)?

Reply:
just to clarify the context for II (I have no context here): II pins cargo & npm which are the two important tools; and npm statically links most libraries that may cause differences in the build output (zlib, etc)

Reply:
ok, looks like all the bash files were taken from II. If reproducibility is important you might also want to copy the repro tests that run on CI:
* https://github.com/dfinity/internet-identity/blob/c31e8e1c2cd0ba2e9b9fa2a5b93f2e26d1d0a433/.github/workflows/canister-tests.yml#L901-L960
* https://github.com/dfinity/internet-identity/blob/main/.github/workflows/release-build-check.yml

and if the `apt` issue actually becomes an issue you're probably better off switching to bazel or nix, but you should be fine

Reply:
This is a problem shared by II, cycles-ledger and now the EVM RPC canister. Eventually we need to address it.